test/tools/ossfuzz/protomutators/YulProtoMutator.cpp

```diff
@@ -145,3 +172,56 @@ unsigned YPM::EnumTypeConverter<T>::enumMin()
 	else
 		static_assert(AlwaysFalse<T>::value, "Yul proto mutator: non-exhaustive visitor.");
 }
+
+template <typename T>
+void YPM::addControlFlow(T* _msg)
+{
+	enum class ControlFlowStmt: unsigned
+	{
+		For = 0,
+		BoundedFor,
+		If,
+		Switch,
+		FunctionCall,
+		Break,
+		Continue,
+		Leave,
+		Termination
+	};
+	uniform_int_distribution<unsigned> d(
+		static_cast<unsigned>(ControlFlowStmt::For),
+		static_cast<unsigned>(ControlFlowStmt::Termination)
+	);
+	auto random = static_cast<ControlFlowStmt>(d(s_rand.m_random));
+	Statement* s = _msg->mutable_block()->add_statements();
```

Comment:
So this is mainly what needs to be changed if we don't do the `block` refactoring?

Reply:
Yup, avoids a type if-else block